tests/notification.spec.ts

```diff
@@ -64,10 +89,10 @@ test.describe('/api/notification', () => {
 
         const newEvent = await strapiInstance.entityService.create('api::event.event', {
             data: {
-                title: 'Testing User Access to Notifications',
                 content: 'Testing User Access to Notifications',
                 entityType: "IdeaCard",
                 entityId: testId,
+                action: "Test",
```

Comment:
ideally we could remove this at a later date when we no longer need to test this feature